src/yogeshNimbalkar/A23_ArrangeLetters1.java

```diff
@@ -0,0 +1,39 @@
+package yogeshNimbalkar;
+
+import java.util.Scanner;
+
+// Return a string by placing all odd numbers , first capital letters from given string, even numbers.
+public class A23_ArrangeLetters1 {
+
+	String getArrangedStr(String str) {
+		String capletter="";
+		String oddNumber="";
+		String evenNumber="";
+		boolean flag = true;
+		for(int index=0; index<str.length(); index++) {
+			char ch = str.charAt(index);
+			if(Character.isDigit(ch)) {
+				if(Character.getNumericValue(ch)%2 == 0) {
+					evenNumber += ch;
+				}else {
+					oddNumber +=ch;
+				}
+			}else if(Character.isLetter(ch)) {
```

Comment:
https://github.com/KrishnaKTechnocredits/JAVATechnoDec2021/pull/907/files

Check the above code, it also handles 2 and 3 digits